celavi/compute_locations.py

```diff
@@ -765,23 +765,8 @@ def capacity_projections_solar(self):
             inplace=True
         )
 
-        # Take sequential differences in installed capacity within each state to
```

Comment:
Why was all that removed? It does not seem to relate to closed loop pathway implementation...